apps/CommonApp/src/release_tests/mouseButtons/index.tsx

```diff
@@ -27,6 +27,7 @@ function Test({ name, gestureHandlers }: TestProps) {
         ]}>
         {gestureHandlers.map((handler, index) => {
           return (
+            // eslint-disable-next-line @eslint-react/no-array-index-key
```

Comment:
I feel like we can disable this for the test apps - I'm pretty sure all of our lists are static, so using index as the key should be fine.

Reply:
Actually I've turned this rule into warning, so this comment is not necessary. Removed in [af1663e](https://github.com/software-mansion/react-native-gesture-handler/pull/3503/commits/af1663e2746be9747ed049fa0f891327e7a3fe09)